src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -136,12 +136,49 @@ public function compile($path = null)
                 $contents .= "<?php /**PATH {$this->getPath()} ENDPATH**/ ?>";
             }
 
+            if ($this->app['config']['view.removeWhitespace'] == true) {
```

Comment:
please use underscore casing on "removeWhitespace" for consistency with existing configuration.

I believe we'll also want to use a strict equality here, so `===`, so not all truthy values are accepted.